sdk/python/kfp/compiler/_read_write_test_config.py

```diff
@@ -58,13 +58,14 @@
             'concat_message',
             'dict_input',
             'identity',
-            'input_artifact',
```

Comment:
I think this should not be removed. Looks like maybe this branch was a continuation of the original cuj3 placeholder branch instead of a branch off of the latest upstream/master?

Reply:
Yes, thanks for the reminder! This happens because of a typo during editing.